html/index.html

```diff
@@ -10,25 +10,31 @@
 </head>
 <body>
     <header class="header">
-        <div class="header__container row">
+        <div class="header__container row-fluid">
+            <input type="checkbox" id="header-menu-checkbox" class="header__container-menu-checkbox" />
             <div class="header__container-main-btn">Landing page</div>
             <div class="header__container-items">
                 <div class="header__container-item">home</div>
                 <div class="header__container-item">features</div>
                 <div class="header__container-item">developers</div>
                 <div class="header__container-item">pricing</div>
                 <div class="header__container-item">contact</div>
-                <div class="header__container-email-item">e-mail: info@domain.com</div>
+                <div class="header__container-item header__container-email-item">e-mail: info@domain.com</div>
             </div>
+            <label class="header__container-menu-btn" for="header-menu-checkbox">
+                <i class="material-icons">
+                    menu
+                </i>
+            </label>
         </div>
     </header>
     <div class="landing">
-        <h2 class="landing__title">LAND YOUR SERVICES IN STYLE</h2>
+        <h2 class="title__white">LAND YOUR SERVICES IN STYLE</h2>
```

Comment:
Don't put CAPS to your HTML, use dedicated CSS property for that.

Reply:
I always do that, that one slipped by.

Reply:
Fixed